### DISCUSSION TITLE
Help: How to make keybinds focus and then fullscreen a window?

### DISCUSSION THREAD

**Discussion by Z-xus** [i3/i3#6169] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/5b521c35-44a4-4b87-98c6-23641072cf58)
This is my usual setup, I need to do a lot of hopping in between neovide, terminals and vscode. I think pressing MOD+V to open focus and then fullscreen VSCode would be pretty nice, maybe a shell script would work..

**Comment by okraits** [comment, marked as answer]
You can do this in your i3 config, by mapping several commands to one keybinding: https://i3wm.org/docs/userguide.html#command_chaining

Example:

```
bindsym $mod+V focus down;fullscreen enable
```

If you had fullscreened another window before pressing that keybinding, you might want to add `fullscreen disable` at the beginning of that command chain.

**Reply by Z-xus** [reply]
Great! Thanks a lot.

**Comment by orestisfl** [comment]
You can use command criteria to fullscreen a specific window. For example,

```
bindsym $mod+v [class="Code"] fullscreen enable
```

This will also make it focused
